docs/development/API/Dshot.md

```diff
@@ -39,19 +41,18 @@ Resulting in a **16 bit** (2 byte) **frame** with the following structure:
 SSSSSSSSSSSTCCCC
 ```
 
-The interesting part is, that 1 and 0 in the Dshot frame are distinguished by their high time. This means that every bit has a certain (constant) length, and the length of the high part of the bit dictates if a 1 or 0 is being received.
+The interesting part is, that 1 and 0 in the DShot frame are distinguished by their high time. This means that every bit has a certain (constant) length, and the length of the high part of the bit dictates if a 1 or 0 is being received.
 
-<img src="dshot_bit.webp" alt="Dshot high times" width="400"/>
-![Dshot high times](./dshot_bit.webp)
+![DShot high times](./dshot_bit.webp)
 
 This has two benefits:
 
 1. Every frame has exactly the same, easy to calculate duration: **16 x (bit period time)**
-2. The measurement of a bit can always be triggered on a rising flank and stopped on a falling flank (or the other way around in case of the inverted signal with [bidirectional Dshot](#bidirectional-dshot))
+2. The measurement of a bit can always be triggered on a rising flank and stopped on a falling flank (or the other way around in case of the inverted signal with [bidirectional DShot](#bidirectional-dshot))
 
```

Comment:
may need `<br/>` here as well
![Uploading image.png…]()

Reply:
Yes, I have been trying to add a line break here as well.